source/common/crypto/utility.h

```diff
@@ -65,7 +66,7 @@ class Utility {
   static PublicKeyPtr importPublicKey(const std::vector<uint8_t>& key);
 
 private:
-  static const EVP_MD* getHashFunction(const absl::string_view& name);
+  static const EVP_MD* getHashFunction(const absl::string_view name);
```

Comment:
There's no value in making this `const absl::string_view`, should just be `absl::string_view` as it's pass-by-value.